package.json

```diff
@@ -245,6 +251,7 @@
         "subPermissions": [
           "configuration.all",
           "finance.expense-classes.collection.get",
+          "module.orders.enabled",
```

Comment:
actually I'm not sure about this, it's settings page

Reply:
IMO not needed. If you set only settings perms for the user, he will able to see orders app icon

Reply:
yes, but for example in ui-orgs, you can't turn on settings separately - it will include module as well with basic view permissions

Reply:
https://docs.google.com/spreadsheets/d/1IJnlWvi4k-wFatDNimx9fQ375evHSIIxtGMookHvs-I/edit#gid=443204981